src/Dompdf.php

```diff
@@ -415,6 +415,24 @@ public function load_html($str, $encoding = null)
         $this->loadHtml($str, $encoding);
     }
 
+    public function loadDOM($doc, $quirksmode = false) {
+        $this->saveLocale();
```

Comment:
Should we save the locale as part of this method? I don't think it's necessary.

Reply:
You're right, that doesn't seem to be necessary